CHANGELOG.md

```diff
@@ -7,11 +7,15 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Enhancements
 
-* None.
+* Add support for additional authentication-related errors that might be
+  reported by a Realm Object Server.
+* An error handler can now be registered upon `{RLM}SyncUser`s in order to
+  report authentication-related errors that affect the user.
 
 ### Bugfixes
 
-* None.
+* Sync users are now automatically logged out upon receiving certain types
+  of errors that indicate they are no longer logged into the server.
```

Comment:
Maybe it'd be worth saying _when_ that can happen here? Not immediately obvious why a user wouldn't be logged in anymore on the server, but assumed as logged in locally.